packages/blade/src/tokens/theme/overrideTheme.stories.tsx

```diff
@@ -0,0 +1,196 @@
+import type { Meta } from '@storybook/react';
+import { Title, Subtitle, Primary, ArgsTable, Stories, PRIMARY_STORY } from '@storybook/addon-docs';
+import type { ReactElement } from 'react';
+import outdent from 'outdent';
+import { Highlight, Link } from '@storybook/design-system';
+import overrideTheme from './overrideTheme';
+import paymentTheme from './paymentTheme';
+import { Button } from '~components/Button';
+import { BladeProvider } from '~components/BladeProvider';
+import { Checkbox, CheckboxGroup } from '~components/Checkbox';
+import Box from '~components/Box';
+import { Text, Title as BladeTitle } from '~components/Typography';
+
+const Page = (): ReactElement => {
+  return (
+    <>
+      <Title />
+      <Subtitle>
+        overrideTheme takes two themes, baseThemeTokens & overrides and returns a new ThemeTokens
+        object, which you can pass into BladeProvider.
+      </Subtitle>
+      <Link href="https://github.com/razorpay/blade/blob/master/packages/blade/src/tokens/theme/overrideTheme.stories.tsx">
+        Link to source example
+      </Link>
+      <br />
+      <br />
+      <Title>Usage</Title>
+      <Highlight language="tsx">{outdent`
+        const customTheme = overrideTheme({
+          baseThemeTokens: paymentTheme, // theme to override
+          overrides: {
+            colors: {
+              onLight: {
+                brand: {
+                  primary: {
+                    '500': 'red',
```

Comment:
Lets pass some `hsla` color and not `red`. We should encourage people to use hsla just like we've used for our own colors